### PR TITLE
minor changes - adding styles

### PR DESCRIPTION
**Pull request by ArCheeq**

### CODE REVIEW

File: src/components/CustomNavLink.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { NavLink } from "react-router-dom";
+
+function CustomNavLink({label, path}) {
+    return (
+        <NavLink className={({isActive}) => "font-bold uppercase " + (isActive ? "text-primary-200" : "")} to={path}>
```
Comment (Misha999777) [inline src/components/CustomNavLink.js:5]: Let's write `className` and `to` on separate lines to make it more readable

**Review by Misha999777 — APPROVED** [review]
LGTM, except for minor comments

**Review by AnastasiiaMjlxy — APPROVED** [review]

**Review by AnastasiiaMjlxy — APPROVED** [review]

File: src/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { NavLink } from "react-router-dom";
+import { useAuth } from "react-oidc-context";
+
+import CustomNavLink from "./CustomNavLink";
+
+function Header() {
+    const { isAuthenticated, signinRedirect } = useAuth();
+
+    const handleClick = (e) => {
```
[the remark this answers is not in the document]
Comment (ArCheeq) [reply src/components/Header.js:9]: It's a habit to use arrow functions, nothing more. I will rename the function

File: src/components/Header.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    <header className="bg-bg-100 mb-[60px]">
+      <div className="container px-6 py-5">
+        <nav className="flex justify-between items-center">
+          <CustomNavLink label="Логотип" path="/" />
+          <CustomNavLink label="Оголошення" path="/posts" />
+          <CustomNavLink label="Про нас" path="/aboutUs" />
+          {isAuthenticated ? (
+            <CustomNavLink label="Профіль" path="/userProfile" />
+          ) : (
+            <a
```
[the remark this answers is not in the document]
Comment (ArCheeq) [reply src/components/Header.js:24]: From a semantic point of tag a is intended to go to another page, but if we do not have a specific link is still better to use the button, right?

File: package.json [hunk 4]
```diff
@@ -36,5 +34,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "tailwindcss": "^3.3.1"
```
[the remark this answers is not in the document]
Comment (ArCheeq) [reply package.json:39]: I used the tailwind documentation to set it up, but I completely agree with you. I'll fix it.

File: src/styles/index.scss [hunk 5]
```diff
@@ -1,6 +1,15 @@
-// Customization
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700&display=swap');
```
[the remark this answers is not in the document]
Comment (ArCheeq) [reply src/styles/index.scss:1]: I've never heard that you can add fonts with npm, I'll know)

File: src/components/CustomNavLink.js (hunk 1 above)
Comment (ArCheeq) [reply src/components/CustomNavLink.js:5]: forgot to use prettier, I'll fix it

**Pull request merged**